scopesim/effects/detector_list.py

```diff
@@ -360,3 +430,72 @@ def __init__(self, pixel_size, x, y, width, height=None, angle=0, gain=1,
         }
 
         super().__init__(array_dict=array_dict, **params)
+
+
+class DetectorList3D(DetectorList):
+    """Pseudo-detector for simple IFU mode.
+
+    .. versionadded:: PLACEHOLDER_NEXT_RELEASE_VERSION
+    """
+
+    dims = "xyz"  # 2 spatial dimensions, 1 spectral dimension
```

Comment:
Maybe a different letter for the spectral dimension? E.g.
```suggestion
    dims = "xyλ"  # 2 spatial dimensions, 1 spectral dimension
```

Reply:
Eww, non-ASCII character in code 😄

Reply:
You are the one adding `# -*- coding: utf-8 -*-` to every file, I'm just following your lead! :yum: